components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -331,7 +331,10 @@ const HostVirtualCards = props => {
         <P>
           <FormattedMessage
             id="Host.VirtualCards.List.Description"
-            defaultMessage="You can now manage and distribute Virtual Cards created on Privacy.com directly on Open Collective. You can assign multiple virtual cards to one collective. Virtual Cards enable quicker transactions, making disbursing money a lot easier!"
+            defaultMessage="You can now manage and distribute Virtual Cards created on Privacy.com directly on Open Collective. You can assign multiple virtual cards to one collective. Virtual Cards enable quicker transactions, making disbursing money a lot easier! {learnMoreLink}"
+            values={{
+              learnMoreLink: <a href="https://docs.opencollective.com/help/fiscal-hosts/virtual-cards">Learn more</a>,
```

Comment:
We need to use a link tag as we do in the message-id `Host.TOSCheckbox` otherwise _Learn more_ wouldn't be translated.

Reply:
@kewitz : Ops, sorry, I missed this one. I've corrected it here and on the Host settings page as well. Hope it looks good now. 😄